projects/mobile-http/src/lib/connections/cordova-http.connection.ts

```diff
@@ -146,9 +146,44 @@ export class CordovaHttpConnection implements Connection {
             throw new Error('The selected responseType is not supported');
         }
       }
-
-      options.data = this.request.getBody();
-
+      switch (this.request.contentType) {
+        // ContentType.JSON
+        case 1:
+          options.serializer = 'json';
+          options.data = this.request.json();
+          options.contentType = 'application/json';
+          break;

```

Comment:
Add one empty line after each break.
```suggestion
          break;

```